ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -1236,3 +1236,82 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: "12h"
+  labels:
+    job-env: aws
+    job-test: e2e
+    job-release: '4.2'
+  name: canary-release-openshift-origin-installer-e2e-aws-4.2-cnv
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-aws-cnv-cluster-profile
```

Comment:
@rmohr pretty sure this is why it was hanging.  the path here needs to align w/ your job name (e2e-aws-cnv)